Procfile.dev

```diff
@@ -0,0 +1,3 @@
+web: rails s -p 3000 -b '0.0.0.0'
+worker: bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
I think we can remove sidekiq for development as there will no corn feature for blog project.

May be remove it completely?(`gems, redis`) 🤔  . What do you think?

Reply:
Good point here! That we remove websockets and make the project lighter :)

Reply:
@junan @malparty I have removed sidekiq, redis, and action cable from the project in [7bc38c0](https://github.com/nimblehq/rails-view-component-blog/pull/18/commits/7bc38c0ec7dc17c3c22eb81ffb99ad67d2506ba7)